code/modules/clothing/head/misc_special.dm

```diff
@@ -76,6 +76,10 @@
 
 /obj/item/clothing/head/hardhat/cakehat/is_hot()
 	return on * heat
+	
+/obj/item/clothing/head/hardhat/cakehat/get_dismemberment_chance()

```

Comment:
err, shouldn't there be some kind of var or flag for this? It can't be entirely force based right?

Either that or I am hard of memory